lib/samson/console_extensions.rb

```diff
@@ -0,0 +1,19 @@
+module Samson
+  module ConsoleExtensions
+    def login(role = Role::SUPER_ADMIN)
+      user = User.where(role_id: role.id).last!
```

Comment:
So you'd wind up as whatever user happened to be the last super admin that was created?  That doesn't sound right.